src/frontend/js/components/CourseRunEnrollment/index.spec.tsx

```diff
@@ -207,8 +227,6 @@ describe('<CourseRunEnrollment />', () => {
       enrollmentsDeferred.resolve([]);
     });
 
-    const button = screen.getByRole('button', { name: 'Enroll now' });
-    expect(button).toHaveAttribute('aria-disabled', 'true');
-    screen.getByText('Sign up or log in to enroll');
+    const button = screen.getByRole('button', { name: 'Log in to enroll' });
```

Comment:
```js
const link = screen.getByTitle('Log in to enroll');
```

Reply:
@jbpenrath why not this? I think `getByRole` is preferred as it ensures the expected role for the element in addition to the content

```
screen.getByRole('link', { name: 'Log in to enroll' })
```